### PR TITLE
nav component updated

### PR DESCRIPTION
**Pull request by HarshithaKonduru214**

### CODE REVIEW

**Comment by HarshithaKonduru214** [comment]
Pls review the navigation component. I have attached the screenshot below and let me know any changes below.
image

File: Components/nav/nav.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    <link rel="stylesheet" href="/style.css">
+    <link rel="stylesheet" href="nav.css">
+</head>
+<body>
+    <nav class="navbar margin-bottom padding-top flex-row box-shadow">
+		<div class="nav-title align-center-item">OXYGEN UI</div>
+		<div class="nav-buttons flex-row">
+			<a href="https://github.com/HarshithaKonduru214" target="__blank" class="nav-icon button margin"><img src="/Documentation/images/github-brands.svg" /></a>
+			<a href="https://twitter.com/HarshitaKonduru" target="__blank" class="nav-icon button margin"><img src="/Documentation/images/twitter-brands.svg" /></a>
+			<a href="https://www.linkedin.com/in/harshita-k/" target="__blank" class="nav-icon button margin"><img src="/Documentation/images/linkedin-brands.svg" /></a>
```
Comment (Maianki) [inline Components/nav/nav.html:17]: Use alt attributes in images.

**Review by Maianki — APPROVED** [review]
Suggested a minor change. Everything else looks good.

**Review by ayush65 — APPROVED** [review]
Everything looks good

**Review by NtshVrm — APPROVED** [review]
Good job👏

**Review by dcs-soni — APPROVED** [review]
Looks good to merge 👍 Just a suggestion: You can give some margin to the `OXYGEN UI` as I can see there is no space on the left side of the logo. Or else left padding would also work.

File: Components/nav/nav.html (hunk 1 above)
Comment (HarshithaKonduru214) [reply Components/nav/nav.html:17]: Yes I will update it.

**Pull request merged**